src/components/teaser/teaser.scss

```diff
@@ -57,17 +61,15 @@
   flex-flow: var(--sbb-teaser-flex-direction) nowrap;
   align-items: var(--sbb-teaser-align-items);
   gap: var(--sbb-teaser-gap);
-  height: var(--sbb-teaser-container-height, auto);
 }
 
 ::slotted([slot='image']) {
   will-change: transform;
   display: block;
-  object-fit: cover;
```

Comment:
When we have to provide default width and aspect ratio I guess we need also the object-fit: cover? and because in width and aspect ratio there are vars, this could be migrated the same :)

Reply:
yes, in order to prevent image stretching, we should keep `object-fit: cover;`